gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=0.6.3-SNAPSHOT
-ballerinaLangVersion=2.0.0-Preview6
+ballerinaLangVersion=2.0.0-Preview7-SNAPSHOT
 stdlibIoVersion=0.5.2
```

Comment:
We have to update the dependent module versions. @MadhukaHarith92 shall we send a new PR with that?

Reply:
Ack. Will send.